### ISSUE TITLE
Apache2 full version - access to mariadb from php code

### ISSUE THREAD

**Issue by jayo3** [FaserF/hassio-addons#80]
**Describe the bug**
I can not get a connection to the database using php code in a web page.

**To Reproduce**

inside container using bash :
mysql -u root 
ERROR 2002 (HY000): Can't connect to local server through socket '/run/mysqld/mysqld.sock' 

**Expected behavior**
I expect to be able to connect to the mariadb and fetch data to/from it from php in a web server/page

**What Addon?**
 - Apache2
- Addon Version:  2.4.1

**Additional context**
Error reported: "Failed to connect to database: could not find driver"

**Comment by FaserF** [comment]
Are you using the MariaDB Addon as described here?
https://github.com/FaserF/hassio-addons/blob/master/apache2/README.md#installation

I will try to improve the readme about MariaDB integration.

this Apache server just comes with the php modules to speak with the HA Maria DB Addon, it is not configured to communicate with another database or with a local database within the container.

**Comment by jayo3** [comment]
Yes, I was using the addon, the full version.  I am removing all and reinstalling them to see if some connection is then repaired.  I was able to see the database with phpMyAdmin.  Thanks for clarification that there is not another instance of the db inside the docker container for Apache2.Jay    From: Fabian SeitzSent: Saturday, July 30, 2022 5:15 PMTo: FaserF/hassio-addonsCc: Jay Offutt; AuthorSubject: Re: [FaserF/hassio-addons] Apache2 full version - access to mariadb from php code (Issue #80) Are you using the MariaDB Addon as described here?https://github.com/FaserF/hassio-addons/blob/master/apache2/README.md#installationI will try to improve the readme about MariaDB integration.this Apache server just comes with the php modules to speak with the HA Maria DB Addon, it is not configured to communicate with another database or with a local database within the container.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by FaserF** [comment]
Great to hear that. Will be waiting for your feedback.

**Comment by jayo3** [comment]
Got it working - had to have Mariadb use port 3306 and in the connect request in the php code on the website specify the IP address of the computer (not localhost) and also the 3306 port.  After that - got connection and data :) 
Thanks!
